contrib/lib/src/lib.rs

```diff
@@ -21,6 +21,7 @@
 //! * [handlebars_templates](struct.Template.html)
 //! * [tera_templates](struct.Template.html)
 //! * [uuid](struct.Uuid.html)
+//! * [databases](databases/index.html)
```

Comment:
I _think_ you can just use `databases` as the URL and rustdoc will link it to the module.

Reply:
Alright, I'll try that out.

Reply:
Did that work?